README.md

```
@@ -92,14 +93,24 @@ This app uses electron-builder to package and sign the app. Run this command to
 npm run package
 ```
 
+## Install using winget
+To install the desktop app using winget on Windows, type below in the Windows Terminal
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)